great-idea/index.html

```diff
@@ -18,6 +18,82 @@
 
     <!-- Copy and paste your HTML from the first UI project here -->
       
-    
+    <div class="container">
+      <header>
+      <nav>
+        <a href="services.html">Services</a>
+        <a href="#">Product</a>
+        <a href="#">Vision</a>
+        <a href="#">Features</a>
+        <a href="#">About</a>
+        <a href="#">Contact</a>
+       
+        <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+        
+      </nav>
+      </header>
+        
+  
+      <section class="contents">
+          <div class="top-left">
+      <h1 class="index">Innovation<br>
+              On<br>
+              Demand</h1>
+            
+              <a class="btn" href="#">Get Started</a>
+            </div>
+            <div class="top-right">
+              <img src="img/header-img.png" alt="Image of a code snippet.">
+            </div>
+      </section>
+      
+    <hr>
+    <section class="top-content">
```

Comment:
this is an example of the type of names to use for your class its really direct on what its named